yaksh/templates/yaksh/courses.html

```diff
@@ -313,12 +495,151 @@ <h4>{{course.creator.get_full_name.title}}</h4>
     {% endfor %} <!-- end for courses -->
     </tr>
     </table>
-</div>
+    </div>
 {% endif %}
 </div>
 {% endif %}
 <!-- End if all Courses  -->
 
+<!-- ***********************Show Only Active Allotted Courses******************* -->
```

Comment:
The same table is shown for active allotted courses and my active. This is redundant